### PR TITLE
created todolist with ts and material UI

### PR DESCRIPTION
**Pull request by shaileshdeuex**
Added functionality of add tsak and remove task

### CODE REVIEW

File: src/component/Add.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  root: {
+    "& > *": {
+      margin: theme.spacing(1),
+    },
+  },
+}));
+
+type AddProps = {
+  inputTask: string;
+  setInputTask: Function;
```
Comment (yug-shah0106) [inline src/component/Add.tsx]: instead of function use () => returnType

File: src/component/Add.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  handleSubmit: Function;
```
Comment (yug-shah0106) [inline src/component/Add.tsx]: same here

File: src/App.tsx [hunk 3]
```diff
@@ -1,10 +1,69 @@
...
+  return Math.floor(Math.random() * Math.floor(1000000));
+}
+
+export type Todo = {
+  id: number;
+  task: string;
+}[];
 
 function App() {
+  const defaultTodos: Todo = [
```
Comment (yug-shah0106) [inline src/App.tsx]: Todo : {
  id: number;
  task: string;
}
defaultTodos: Array<Todo>

File: src/App.tsx [hunk 4]
```diff
@@ -1,10 +1,69 @@
...
+  };
+
   return (
-    <div className="app">
-      Hello TypeScript
-    </div>
+    <Container className="app" fixed>
+      <Add
+        inputTask={inputTask}
+        setInputTask={setInputTask}
```
Comment (yug-shah0106) [inline src/App.tsx]: (e) => {
setInputTask(e.target.value)
}

File: src/component/Add.tsx (hunk 1 above)
Comment (yug-shah0106) [reply src/component/Add.tsx]: () => void
(input:string) => string
(input:string) => Promise<void> for async function

File: src/component/Add.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      <TextField
+        onChange={(e) => setInputTask(e.target.value)}
+        value={inputTask}
+        id="outlined-basic"
+        label="Outlined"
+        variant="outlined"
+      />
+      <Button
+        type="submit"
+        onClick={(e) => handleSubmit(e)}
```
Comment (yug-shah0106) [inline src/component/Add.tsx]: onCLick :{handleSubmit}

File: src/component/Display.tsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+type DisplayProps = {
+  tasks: Todo;
+  deleteTask: Function;
+};
+
+function Display({ tasks, deleteTask }: DisplayProps) {
+  return (
+    <div className="task_container">
+      {tasks.length === 0 ? (
```
Comment (yug-shah0106) [inline src/component/Display.tsx]: !task.length ?

File: src/App.tsx [hunk 7]
```diff
@@ -38,7 +38,10 @@ function App() {
 
   const handleSubmit = (event: React.MouseEvent<HTMLButtonElement>) => {
     event.preventDefault();
-    console.log("Click");
+    if (!inputTask) {
+      alert("You can't leave inpute blank");
```
Comment (shahsank3t) [inline src/App.tsx]: You are using material ui so use some component from it to show error message and not alert.

**Review by shahsank3t — APPROVED** [review]
looks good to me!

**Pull request merged**